### PR TITLE
DEV: Add Auxiliary Functionality For SlideGraph

### PR DESCRIPTION
**Pull request by vqdang**
Migrate existing .py in #180 to another branch. Included functionalities are:
- Generic graph plotting onto canvas
- Plat Scaler for calibrating probabilities
- Pin scikit-image to avoid errors due to changes in the updated version.

- [x] Add functionalities
- [x] Add tests

### CODE REVIEW

File: tiatoolbox/tools/scale.py [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software Foundation,
+# Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+#
+# The Original Code is Copyright (C) 2021, TIA Centre, University of Warwick
+# All rights reserved.
+# ***** END GPL LICENSE BLOCK *****
+
+import numpy as np
```
Comment (shaneahmed) [inline tiatoolbox/tools/scale.py:21]: need a docstring for this file.

**Review by shaneahmed — COMMENTED** [review]
I only have one comment on this PR as mentioned in scale.py
For travis, can you try to PIN opencv to the previous version on which it passed? I think that might be the issue.

File: tiatoolbox/tools/scale.py (hunk 1 above)
Comment (vqdang) [reply tiatoolbox/tools/scale.py:29]: resolved in https://github.com/TissueImageAnalytics/tiatoolbox/pull/208/commits/d87bc375e7213145d227d8005ebc37802b683b4c

**Review by shaneahmed — APPROVED** [review]

**Comment by John-P** [comment]
We now have some duplicate functionality to resolve. There is visualisation as part of the graph construction class and also another graph plotting function.

**Pull request merged**